astropy/coordinates/tests/test_angles.py

```diff
@@ -1092,3 +1092,54 @@ def test_str_repr_angles_nan(cls, input, expstr, exprepr):
     # Deleting whitespaces since repr appears to be adding them for some values
     # making the test fail.
     assert repr(q).replace(" ", "") == f'<{cls.__name__}{exprepr}>'.replace(" ","")
+
+
+@pytest.mark.parametrize("sign", (-1, 1))
+@pytest.mark.parametrize(
```

Comment:
Do you think this would benefit from using hypothesis or is that overkill?

Reply:
Overkill... It might well find little corner cases that I'd rather be kept hidden...

Reply:
🙈